tests/basic.test.js

```diff
@@ -177,8 +177,46 @@ test('Should get dataset from server', async () => {
         calendarsData    : expect.arrayContaining([expect.objectContaining({
             id        : expect.any(String),
             intervals : expect.anything()
-        })])
+        })]),
+        versionsData     : expect.arrayContaining([expect.objectContaining({
+            id         : expect.any(String),
+            name       : expect.any(String),
+            content    : expect.any(String)
+        })]),
+        changelogData    : expect.any(Array)
     });
 
     ws.terminate();
 });
+
+test('Should receive OK to autosave once', async () => {
+    const ws = new WebSocket(server.address);
+    const ws1 = new WebSocket(server.address);
+    const ws2 = new WebSocket(server.address);
+
+    await waitForConnectionOpen(ws);
+    await waitForConnectionOpen(ws1);
+    await waitForConnectionOpen(ws2);
+
+    await Promise.all([
+        awaitDataset(ws, 1),
+        awaitDataset(ws1, 1),
+        awaitDataset(ws2, 1)
+    ]);
+
+    ws.send('{ "command": "requestVersionAutoSave", "project": 1 }');
+    ws1.send('{ "command": "requestVersionAutoSave", "project": 1 }');
+    ws2.send('{ "command": "requestVersionAutoSave", "project": 1 }');
+
+    const [response1, response2, response3] = await Promise.allSettled([
+        awaitNextMessage(ws),
+        awaitNextMessage(ws1),
+        awaitNextMessage(ws2)
+    ]);
```

Comment:
`awaitNextMessage` has a 2nd argument for the request. it will set up listener for the `message` and then will send the command. I think it should be identical to this:

```suggestion
    const [response1, response2, response3] = await Promise.allSettled([
        awaitNextMessage(ws, { "command": "requestVersionAutoSave", "project": 1 }),
        awaitNextMessage(ws1, { "command": "requestVersionAutoSave", "project": 1 }),
        awaitNextMessage(ws2, { "command": "requestVersionAutoSave", "project": 1 })
    ]);
```